### ISSUE TITLE
Turned off antivirus - Still not working

### ISSUE THREAD

**Issue by Freeo29** [finndev/PokeBuddy#512]
I'm having the same problem as MasterSloth had (Issue #458). I have turned off my antivirus (I have avast on Win 10) tried redownloading but i am unable to, and am still getting notifications saying a threat has been blocked etc. 

I'm not the most technically gifted when it comes to computers so any further help would be greatly appreciated.

Thanks
![3b06c7aa-5cf3-11e6-8171-85d2fd6fad14](https://cloud.githubusercontent.com/assets/20911923/17491956/a886e5c2-5da1-11e6-8d22-2869e11e4a95.png)

![652a018c-5cf3-11e6-8506-54b80d28e102](https://cloud.githubusercontent.com/assets/20911923/17491946/9f941a34-5da1-11e6-9f6e-59dd948c10b3.jpg)

**Comment by curtiola** [comment]
It's still being blocked by a firewall, whether it's windows defender/firewall or an antivirus

**Comment by Freeo29** [comment]
When I turned off Avast it said that Windows firewall was also turned off. There isnt any other firewall/antivirus programs on my computer

**Comment by curtiola** [comment]
It wouldn't block anything it thought had a virus if there wasn't something checking for viruses, maybe you have another one running somewhere lol

**Comment by Freeo29** [comment]
I'm stumped then lol, guess I'll have to play the game normally

**Comment by Razdavitel** [comment]
Perhaps this could be blocked by a router firewall? Or are you running this at work? Has the program worked for you before?

**Comment by curtiola** [comment]
Yeah the router could be the culprit if it's firewall is set higher than normal

**Comment by Seraphim32** [comment]
@curtiola It could also be VS trying to run the program and noticing a modification that would indicate actions such as a virus.
The firewall wouldnt modify the files, just block in/out data (which would not affect local files), nor would the router. Unless you had a stronger firewall that blocks the actual downloaded files from arriving in one piece.

I think more information is needed, as for what information, that's the question.
It looks more like something intercepted the download and it corrupted the file.

**Comment by curtiola** [comment]
That could be, the other people who had this same message just had to disable Antivirus though, but yeah you're probably right, maybe some super nazi Internet security or something haha

**Comment by Seraphim32** [comment]
That would be prior to downloading then (unless they have an intrusive AV that go in and hold the file while scanning) if the AV determines that the download contains infected files. (which might result in the error of the file being tampered with).
And speaking of HW firewalls, gotta check mine so it's not messing with my problem i have with this program. :)
